modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro

```diff
@@ -267,4 +267,99 @@ definition {
 		AssertElementPresent(locator1 = "PRMMDFRequest#ERROR_MESSAGE");
 	}
 
+	@summary = "Verify the fills done by the user on the Campaign Information"
+	macro viewMDFActivityDetails {
+		Click(locator1 = "Form#FIELDS_GROUP_COLLAPSED");
```

Comment:
Hi @MGabriellaS, here you can use an existing path: `PRMMDFRequest#TEXT_BUTTON`, that was created to the same element, but on MDF Review Page. But you can improve the path changing to `//button[contains(@class,'collapse-icon')]//*[contains(.,'${key_textButton}')]`. It will find the collapse on MDF Detail Page and on MDF Review Page